### PR TITLE
Origin/login activity

### PR DESCRIPTION
**Pull request by nir-ya**
pull login activity

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">My Application</string>
```
Comment (avnerduc) [inline app/src/main/res/values/strings.xml]: Maybe we can set a more informative / cooler app name

File: app/src/main/java/com/example/myapplication/LoginActivity.java [hunk 2]
```diff
@@ -39,22 +39,40 @@ protected void onCreate(Bundle savedInstanceState) {
     public void onStart() {
         super.onStart();
         // Check if user is signed in (non-null) and update UI accordingly.
-        FirebaseUser currentUser = mAuth.getCurrentUser();
-        updateUI(currentUser);
+//        FirebaseUser currentUser = mAuth.getCurrentUser();
```
Comment (avnerduc) [inline app/src/main/java/com/example/myapplication/LoginActivity.java]: Can we remove this comment?

File: app/src/main/java/com/example/myapplication/LoginActivity.java [hunk 3]
```diff
@@ -63,31 +81,31 @@ public void onComplete(@NonNull Task<AuthResult> task) {
...
-            Toast.makeText(this, "user logged in", Toast.LENGTH_SHORT).show();
-        }
-    }
+//    private void updateUI(FirebaseUser user) {
+//        if (user == null) {
+//            Toast.makeText(this, "no user", Toast.LENGTH_SHORT).show();
+//        } else {
+//            Toast.makeText(this, "user logged in", Toast.LENGTH_SHORT).show();
+//        }
+//    }
```
Comment (avnerduc) [inline app/src/main/java/com/example/myapplication/LoginActivity.java]: I think it's best not to pull comments

File: app/src/main/java/com/example/myapplication/LoginActivity.java [hunk 4]
```diff
@@ -63,31 +81,31 @@ public void onComplete(@NonNull Task<AuthResult> task) {
...
 
     @Override
     protected void onStop() {
         super.onStop();
 
-        mAuth.signOut();
+//        mAuth.signOut();
```
Comment (avnerduc) [inline app/src/main/java/com/example/myapplication/LoginActivity.java]: here as well

File: app/src/main/java/com/example/myapplication/LoginActivity.java [hunk 5]
```diff
@@ -50,42 +50,37 @@ public void login(View view) {
...
         final String password = passwordText.getText().toString();
 
-        mAuth.signInWithEmailAndPassword(emailAddress, password).addOnCompleteListener(this,
+        firebaseAuth.signInWithEmailAndPassword(emailAddress, password).addOnCompleteListener(this,
                 new OnCompleteListener<AuthResult>() {
                     @Override
                     public void onComplete(@NonNull Task<AuthResult> task) {
                         if (task.isSuccessful()) {
-                            // Sign in success, update UI with the signed-in user's information
-                            Log.d(TAG, "signInWithEmail:success");
```
Comment (avnerduc) [inline app/src/main/java/com/example/myapplication/LoginActivity.java]: Maybe it's better to log things like this

File: app/src/main/java/com/example/myapplication/LoginActivity.java [hunk 6]
```diff
@@ -50,42 +50,37 @@ public void login(View view) {
...
                         if (task.isSuccessful()) {
-                            // Sign in success, update UI with the signed-in user's information
-                            Log.d(TAG, "createUserWithEmail:success");
-                            FirebaseUser user = mAuth.getCurrentUser();
+
                             Toast.makeText(LoginActivity.this, "user signed up",
                                     Toast.LENGTH_SHORT).show();
-                        } else {
-                            // If sign in fails, display a message to the user.
-                            Log.w(TAG, "createUserWithEmail:failure", task.getException());
```
Comment (avnerduc) [inline app/src/main/java/com/example/myapplication/LoginActivity.java]: log maybe?

File: app/src/main/java/com/example/myapplication/LoginActivity.java [hunk 7]
```diff
@@ -94,18 +89,12 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                 });
     }
 
-//    private void updateUI(FirebaseUser user) {
-//        if (user == null) {
-//            Toast.makeText(this, "no user", Toast.LENGTH_SHORT).show();
-//        } else {
-//            Toast.makeText(this, "user logged in", Toast.LENGTH_SHORT).show();
-//        }
-//    }
```
Comment (avnerduc) [inline app/src/main/java/com/example/myapplication/LoginActivity.java]: commented text shouldn't be committed, I understand Alex said

File: app/src/main/java/com/example/myapplication/LoginActivity.java [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+                });
+    }
+
+
+    @Override
+    protected void onStop() {
+        super.onStop();
+
+        // todo sign out?
```
Comment (RanHadar) [inline app/src/main/java/com/example/myapplication/LoginActivity.java]: We need to think about it

File: app/src/main/res/layout/activity_login.xml [hunk 9]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (RanHadar) [inline app/src/main/res/layout/activity_login.xml]: Let's fix a bit the design first before merging

Comment (RanHadar) [reply app/src/main/res/layout/activity_login.xml]: Great Now!

**Comment by avnerduc** [comment]
A job well done

**Pull request merged**